### PR TITLE
adjust the themes, if find a theme in group, no need keep looking for…

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/NP-2737
nanopay side:https://github.com/nanoPayinc/NANOPAY/pull/11174
notes: if we find theme though group, we no longer need find its parent group. In back office case, it will always be nanopay

### CODE REVIEW

**Review by zahnno — APPROVED** [review]

File: src/foam/nanos/theme/Themes.js [hunk 1]
```diff
@@ -177,15 +177,17 @@ Later themes:
...
+        Theme groupTheme = null;
+        while ( groupTheme == null &&  group != null) {
+          groupTheme = group.findTheme(x);
           if ( groupTheme != null ) {
             theme = (Theme) theme.fclone().copyFrom(groupTheme);
             if ( ! SafetyUtil.isEmpty(group.getDefaultMenu()) ) {
               theme.setDefaultMenu(group.getDefaultMenu());
             }
+          } else {
+            group = (Group) groupDAO.find(group.getParent());
```
Comment (lchanmann) [inline src/foam/nanos/theme/Themes.js]: can we just break at after applying groupTheme instead?

Comment (yij793) [reply src/foam/nanos/theme/Themes.js]: like this ?
```java
          if ( groupTheme != null ) {
            theme = (Theme) theme.fclone().copyFrom(groupTheme);
            if ( ! SafetyUtil.isEmpty(group.getDefaultMenu()) ) {
              theme.setDefaultMenu(group.getDefaultMenu());
            }
          break;
}
```

Comment (jlhughes) [reply src/foam/nanos/theme/Themes.js]: yes, just break at the end of the if ( groupTheme != null ) { block }

Comment (jlhughes) [reply src/foam/nanos/theme/Themes.js]: So this should be a one line delta. 
All you need is 'break;' 
You can remove everything else you added.

**Review by jlhughes — APPROVED** [review]

**Pull request merged**